src/Symfony/Component/Mime/HtmlToTextConverter/HtmlToTextConverterInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\HtmlToTextConverter;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+interface HtmlToTextConverterInterface
+{
+    /**
+     * Converts and HTML representation of a Message to a text representation.
+     */
+    public function convert(string $html): string;
```

Comment:
do we need to specify something about the charset, eg "the output must be UTF-8"?

Reply:
See the second commit, I've done something a bit different.